post/clang_tidy_review/clang_tidy_review/__init__.py

```diff
@@ -892,6 +975,19 @@ def create_review(
     # Read and parse the CLANG_TIDY_FIXES file
     clang_tidy_warnings = load_clang_tidy_warnings()
 
+    # Read and parse the timing data
+    clang_tidy_profiling = load_and_merge_profiling()
+
+    try:
+        sha = pull_request.head_sha
+    except Exception:
+        sha = os.environ.get("GITHUB_SHA")
+
+    # Post to the action job summary
+    step_summary = ""
+    step_summary += make_timing_summary(clang_tidy_profiling, sha)
+    set_summary(step_summary)
```

Comment:
Not sure I see the purpose of the first line here?

Reply:
Ok -- that's the sort of thing I think is better to just refactor when needed. Or perhaps take a list of summary items and then we can do `"\n".join(summary_list)`